src/theme-googlecharts.ts

```diff
@@ -0,0 +1,57 @@
+import {Config} from './config';
+
+const markColor = '#3366CC';
+const gridColor = '#ccc';
+const defaultFont = 'Arial, sans-serif';
+
+const googlechartsTheme: Config = {
+  arc: {fill: markColor},
+  area: {fill: markColor},
+  path: {stroke: markColor},
+  rect: {fill: markColor},
+  shape: {stroke: markColor},
+  symbol: {stroke: markColor},
+  circle: {fill: markColor},
+  background: '#fff',
+  padding: {
+    top: 10,
+    right: 10,
+    bottom: 10,
+    left: 10
+  },
+  title: {
+    font: defaultFont,
+    fontSize: 12,
+    dy: -3,
+    anchor: 'start'
+  },
+  axis: {
```

Comment:
Do you also want to set the font for legends? Maybe you can refactor this theme a bit and use guide styles.

Reply:
I do want to set the default font everywhere. I saw the guide styles used in the dark theme. If I set the font within `guide-label`, `guide-title` and `group-title` (and take out the existing font config on title, label) will that be sufficient?

Reply:
I think that's enough.

Reply:
I did a bit of experimenting and it looks like `title { font }` needs to be set as well, just for vega-lite. Vega takes the style from `guide-title`. I went ahead and set the font on all 4 properties, which covered all the usage in the example graphs.